package.json

```diff
@@ -2,7 +2,7 @@
   "name": "building-models",
   "description": "System dynamics modeling tool.",
   "author": "The Concord Consortium",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Should probably update `package-lock.json` as well.

Reply:
:+1: